index.html

```diff
@@ -195,6 +195,88 @@ <h4>Multi-Post Stories</h4>
       </div>
     </section>
 
+    <section class="contacts-area">
+      <div class="contacts">
+        <h6>CONTACT US</h6>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form class="contacts-form" action="#" method="post">
+        <ul class="input-area">
+          <li class="text-area">
+            <label for="name"></label>
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              placeholder="Text Input"
+            />
+          </li>
+
+          <li class="email-area">
+            <label for="mail"></label>
+            <input
+              type="email"
+              id="mail"
+              name="user_email"
+              placeholder="jeremy@gmail.com"
+            />
+          </li>
+
+          <li class="message-area">
+            <textarea
+              name="msg"
+              id="msg"
+              placeholder="Enter text here"
+            ></textarea>
+          </li>
+        </ul>
+      </form>
+
+      <a class="button button-frame" href="#">see Project</a>
```

Comment:
- [ ] Kindly ensure your button, is a `<button> element or an `<input>` element , if you choose `<input>` element ensure  you add type `submit`.Also, ensure the hover effect is aligned to the design given.